package.json

```diff
@@ -92,14 +91,14 @@
         "postcss": "^8.3.6",
         "postcss-load-config": "^3.1.0",
         "prettier": "^2.3.2",
-        "react": "17.0.2",
-        "react-dom": "17.0.2",
+        "react": "^18.3.1",
```

Comment:
Same comment as above re: locking `react` and `react-dom` versions

Reply:
done